docs/001_develop/03_client-capabilities/008_interaction/select.mdx

```diff
@@ -0,0 +1,317 @@
+    ```html
+    <rapid-select>
+      <rapid-option value="s">Small</rapid-option>
+      <rapid-option value="l">Large</rapid-option>
+    </rapid-select>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+        <rapid-select @change=${x => x.selectChanged()}>
+          <rapid-option value="s">Small</rapid-option>
+          <rapid-option value="l">Large</rapid-option>
+        </rapid-select>
+      `,
+    })
+    export class MyElement extends GenesisElement {
+      selectChanged() {
+        alert('Select Changed')
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+  Declaration
+  ```html
+  <rapid-select>
+    <rapid-option value="s">Small</rapid-option>
+    <rapid-option value="l">Large</rapid-option>
+  </rapid-select>
+  ```
+  Usage
+  ```typescript
+  export function MyComponent() {
+    const handleSelectChanged = (event) => {
+      console.log('Select changed')
+    };
+
+    return (
+      <rapid-select onChange={handleSelectChanged}>
+        <rapid-option value="s">Small</rapid-option>
+        <rapid-option value="l">Large</rapid-option>
+      </rapid-select>
+    );
+  }
+  ```
+</TabItem>
+  <TabItem value="angular">
+
+  Declaration
+  ```html
+  <rapid-select>
+    <rapid-option value="s">Small</rapid-option>
+    <rapid-option value="l">Large</rapid-option>
+  </rapid-select>
+  ```
+
+  Usage
+  ```typescript
+  import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+  import { FormsModule } from '@angular/forms';
+  @Component({
+    selector: 'my-root',
+    template: `
+          <rapid-select (change)="onSelectChanged($event)">
+            <rapid-option value="s">Small</rapid-option>
+            <rapid-option value="l">Large</rapid-option>
+          </rapid-select>
+        `,
+    standalone: true,
+    schemas: [CUSTOM_ELEMENTS_SCHEMA],
+    imports: [FormsModule],
+  })
+  export class AppComponent {
+    onSelectChanged(event: Event) {
+      console.log('Select changed');
+    }
+  }
+  ```
+</TabItem>
+</Tabs>
+
+## API
+
+ Property and attribute binding examples for Genesis Component syntax. Closing tag omitted.
+
+<table>
+  <thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Example</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>disabled</td>
+      <td><code>boolean</code></td>
+      <td>Similar to `readonly`, but with a blur on the component.</td>
+<td>
+
+```typescript
+<rapid-select disabled>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>form</td>
+      <td><code>string</code></td>
+      <td>Associates this component with a form. Form `id` needs to be passed. If no Id is provided, then it will be associated with the ancestor form.</td>
+<td>
+
+```typescript
+<rapid-select form="sampleForm">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>length</td>
+      <td><code>number</code></td>
+      <td>The number of options.</td>
+<td>
+
+```typescript
+<rapid-select number=10>
+```
```

Comment:
I was a bit confused about this when I read it. Reading up on it some more I think this has been depreacted in favour of size. Let's just drop this and only mention size
```suggestion
```